torch/_dynamo/convert_frame.py

```diff
@@ -1484,9 +1484,15 @@ def log_bytecode(
                 package=package,
             )
         except exc.SkipFrame as e:
-            if one_graph:
-                log.debug("No graph captured with export/fullgraph=True")
             assert e._torch_dynamo_tracer_output is not None
+            if one_graph:
+                err = Unsupported(
+                    "torch.compile(..., fullgraph=True) was requested, but Dynamo did not capture "
+                    f"a graph for {code.co_name} ({code.co_filename}:{code.co_firstlineno}). "
+                    "Skipped frames are not allowed with fullgraph=True."
+                )
+                err._torch_dynamo_tracer_output = e._torch_dynamo_tracer_output  # type: ignore[attr-defined]
+                raise err from e
```

Comment:
Prefer not using `Unsupported` here - either a new exception class or `unimplemented(...)`.

I'm currently thinking of replace all `unimplemented(...)` calls outside of symbolic_convert with a new exception that signifies a non-graph-break tracing failure. Will keep updated on what I decide to do.